apps/builder/app/builder/shared/style-source/style-source-input.tsx

```diff
@@ -48,6 +48,7 @@ import {
   type ReactNode,
```

Comment:
Why move whole section out of style panel? You could move to shared only element store